content/en/database_monitoring/setup_sql_server/azure.md

```diff
@@ -128,7 +128,7 @@ For [SQL Server on Windows Azure VM][1] follow the [Setting Up Database Monitori
 
 ## Install the Agent
 
-Since Azure does not grant direct host access, the Datadog Agent must be installed on a separate host where it is able to talk to the SQL Server host. There are several options for installing and running the Agent.
+**Because Azure does not grant direct host access, the Datadog Agent must be installed on a separate host where it is able to talk to the SQL Server host**. There are several options for installing and running the Agent.
```

Comment:
I would leave this unbolded here, since we aren't bolding it elsewhere -- I think Casey was just worried about the AlwaysOn verbiage specifically.

Reply:
Okay, happy to defer to you on this one.